api/src/main/java/jakarta/data/repository/Find.java

```diff
@@ -62,12 +62,8 @@
  *     <li>Array of the entity type</li>
  * </ul>
  *
- * <p>If the annotated method return type is a single instance (either through {@code Optional} or directly), but the query returns more than one element,
- * it will throw a {@link jakarta.data.exceptions.NonUniqueResultException}.</p>
- *
- * <p>If this annotation is combined with other operation annotations (e.g., {@code @Update}, {@code @Delete},
- *  {@code @Save}), it will result in an {@link UnsupportedOperationException} being thrown, as only one operation type can be specified.
- *  A Jakarta Data provider implementation must detect and report this error either at compile time or runtime.</p>
+ * <p>If the annotated method return type is a single instance (either through {@code Optional} or directly), but the
+ * query returns more than one element, it will throw a {@link jakarta.data.exceptions.NonUniqueResultException}.</p>
```

Comment:
```suggestion
 * query returns more than one element, it will throw a {@link jakarta.data.exceptions.NonUniqueResultException}.</p>
 * <p>If this annotation occurs alongside {@code @Delete}, {@code @Find}, {@code @Insert},
 * {@code @Query}, {@code @Save}, or {@code @Update}, the annotated repository method must raise
```